website/docs/docs/core/connect-data-platform/risingwave-setup.md

```diff
@@ -0,0 +1,87 @@
+  pypi_package: 'dbt-risingwave'
+  authors: 'Dylan Chen'
+  github_repo: 'risingwavelabs/dbt-risingwave'
+  min_core_version: 'v1.6.1'
+  min_supported_version: ''
+  cloud_support: Not Supported
+  slack_channel_name: '#all-things-risingwave'
+  slack_channel_link: 'https://www.risingwave.com/slack'
+  platform_name: 'RisingWave'
+  config_page: '/reference/resource-configs/no-configs'
+---
+
+:::info Vendor-supported plugin
+
+Certain core functionality may vary. If you would like to report a bug, request a feature, or contribute, you can check out the linked repository and open an issue.
+
+:::
+
+import SetUpPages from '/snippets/_setup-pages-intro.md';
+
+<SetUpPages meta={frontMatter.meta} />
+
+## Connecting to RisingWave with dbt-risingwave
+
+Before connecting to RisingWave, ensure that RisingWave is installed and running. For more information about how to get RisingWave up and running, see the [RisingWave quick start guide](https://docs.risingwave.com/docs/dev/get-started/).
+
+To connect to RisingWave with dbt, you need to add a RisingWave profile to your dbt profile file (`~/.dbt/profiles.yml`). Below is an example RisingWave profile. Revise the field values when necessary.
+
+<File name='~/.dbt/profiles.yml'>
+
+```yaml
+default:
+  outputs:
+    dev:
+      type: risingwave
+      host: 127.0.0.1 
+      user: root
+      pass: ""
+      dbname: dev
+      port: 4566
+      schema: public
+  target: dev
+```
+
+</File>
+
+|Field|Description|
+|---|---|
+|`host`| The host name or IP address of the RisingWave instance|
+|`user`|The RisingWave database user you want to use|
+|`pass`| The password of the database user|
+|`dbname` | The RisingWave database name|
+|`port` | The port number that RisingWave listens on|
+|`schema`| The schema of the RisingWave database|
+
+To test the connection to RisingWave, run:
+
+```bash
+dbt debug
+```
+
+## Materializations
+
+The dbt models for managing data transformations in RisingWave are similar to typical dbt SQL models. In the `dbt-risingwave` adapter, we have customized some of the materializations to align with the streaming data processing model of RisingWave.
+
+|Materializations| Supported|Notes|
+|----|----|----|
+|`table` |Yes |Creates a [table](https://docs.risingwave.com/docs/dev/sql-create-table/). To use this materialization, add `{{ config(materialized='table') }}` to your model SQL files. |
+|`view`|Yes | Creates a [view](https://docs.risingwave.com/docs/dev/sql-create-view/). To use this materialization, add `{{ config(materialized='view') }}` to your model SQL files. |
+|`ephemeral`|Yes| This materialization uses [common table expressions](https://docs.risingwave.com/docs/dev/query-syntax-with-clause/) in RisingWave under the hood. To use this materialization, add `{{ config(materialized='ephemeral') }}` to your model SQL files.|
+|`materializedview`| To be deprecated. |It is available only for backward compatibility purposes. Use `materialized_view` instead.|
```

Comment:
What dbt versions was this supported on?

Reply:
The dbt-core version used by RisingWave is `1.6.0`

Reply:
I just added plugin version info to this row, so that people won't be confused about the statement "to be deprecated".